File: mobile/minibilge_mobile/lib/features/experience/models/experience_mode.dart
enum ExperienceMode {
  family,
  child,
  adult;

  String get apiValue => switch (this) {
    ExperienceMode.family => 'Family',
    ExperienceMode.child => 'Child',
    ExperienceMode.adult => 'Adult',
  };

  static ExperienceMode fromApi(String? value) {
    return switch (value?.trim().toLowerCase()) {
      'child' => ExperienceMode.child,
      'adult' => ExperienceMode.adult,
      _ => ExperienceMode.family,
    };
  }
}

File: backend/MiniBilge.Domain/Enums/ExperienceMode.cs
namespace MiniBilge.Domain.Enums;

/// <summary>
/// Controls which product experience is emphasized for the signed-in account.
/// Family is the backward-compatible default for existing MiniBilge accounts.
/// </summary>
public enum ExperienceMode
{
    Family = 0,
    Child = 1,
    Adult = 2
}
